critiquebrainz/frontend/external/bookbrainz_db/edition_group.py

```diff
@@ -0,0 +1,77 @@
+import uuid
+from brainzutils import cache
+from typing import List
+import sqlalchemy
+import critiquebrainz.frontend.external.bookbrainz_db as db 
+from critiquebrainz.frontend.external.bookbrainz_db import DEFAULT_CACHE_EXPIRATION
+from critiquebrainz.frontend.external.bookbrainz_db.identifiers import fetch_identifiers
+from critiquebrainz.frontend.external.bookbrainz_db.relationships import fetch_relationships
+
+
+def get_edition_group_by_bbid(bbid: uuid.UUID) -> dict:
+    """
+    Get info related to an edition group using its BookBrainz ID.
+    Args:
+        bbid : BBID of the edition group.
+    Returns:
+        A dictionary containing the basic information related to the edition group.
+    """
+    edition_group = fetch_multiple_edition_groups([bbid])
+    if not edition_group:
+        return None
+    return edition_group[bbid]
+
+
+def fetch_multiple_edition_groups(bbids: List[uuid.UUID]) -> dict:
+    """
+    Get info related to multiple edition groups using their BookBrainz IDs. 
+    Args:
+        bbids (list): List of BBID of edition groups.
+    Returns:
+        A dictionary containing info of multiple edition groups keyed by their BBID.
+    """
+    if bbids == []:
+        return {}
+
+    key = cache.gen_key('edition-groups', bbids)
+    edition_groups = cache.get(key)
+    if not edition_groups:
+        with db.bb_engine.connect() as connection:
+            result = connection.execute(sqlalchemy.text("""
+                SELECT 
```

Comment:
Take a look a the (in-progress) MusicBrainz SQL formatting guidelines, and see if you can update this query to follow that format: https://gist.github.com/reosarevok/2d8a2fa6310cc0144cad043a9c566717 

eventually we would like to follow this format with all of our queries in all apps, this is a good place to start! Ask me or @reosarevok if you have any questions with the guidelines.